### DISCUSSION TITLE
redirect to github button on home page

### DISCUSSION THREAD

**Discussion by EvertsRozz** [dailydotdev/daily#1103] [Feature Requests]
Add a button that redirects the user to their GitHub page; preferably on the top right side (for example next to or inside the account tab).

Right now, it’s easier to get to GitHub using the search bar instead of going to the ‘/`username`’ path and then hitting the GitHub button. A small life improvement.

**Comment by idoshamun** [comment]
Hi @EvertsRozz,

Just to be clear, do you want a button to make it easy for you to navigate to your own github or any other user on daily.dev?

**Reply by EvertsRozz** [reply]
Hi @idoshamun ,

Sorry for the confusion, a button to navigate to your own github account is what I meant.

The github button in the account tab is, from my experience and understanding, there for other people to click on when visiting my daily.dev account. However it feels unintuitive to have to go to that tab when I want to quickly go to my own github. It would be like a minor ease of use improvement.

Cheers

**Reply by idoshamun** [reply]
Have you considered using our shortcuts feature on the extension?

**Reply by EvertsRozz** [reply]
Sorry for the delay,
Unluckily I'm already using 8 shortcuts. It would be a nice feature if the ability to be redirected to a personal Github account were an out-of-the-box thing for a developer-oriented page.
